src/systems/performer_detector/PerformerDetector.hh

```diff
@@ -67,6 +67,9 @@ namespace systems
   /// `<geometry>`: Detection region. Currently, only the `<box>` geometry is
   /// supported. The position of the geometry is derived from the pose of the
   /// containing model.
+  /// `<pose>`: Additional pose offset. This pose is added to the parent
```

Comment:
Could you add the frame relative to which this pose is given?

Reply:
Add: e34d3bda28d79ebbbc82a8d473e0452b2069971f